### PR TITLE
Minor fcl file fixes

### PR DESCRIPTION
**Pull request by PawelPlesniak**
The previous directory for the STM pileup scripts was `Production/JobConfig/pileup`, it's now `Production/JobConfig/pileup/STM`

### CODE REVIEW

File: Scripts/gen_BeamToVD101.sh [hunk 1]
```diff
@@ -70,8 +70,8 @@ for dirname in 000 001 002 003 004 005 006 007 008 009; do
 done
 
 # Cleanup
-echo "Removing produced files"
-rm -f beamToVD101Resampler.fcl
-rm -f EleBeamCat.txt
-rm -f MuBeamCat.txt
-echo "Finished"
+#echo "Removing produced files"
```
Comment (kutschke) [inline Scripts/gen_BeamToVD101.sh:73]: Just checking: did you mean to leave this commented out?   Unless commented out code is debug code or code under development we would like it to be removed.

**Review by kutschke — APPROVED** [review]
Looks good .  One stylist comment.

File: Scripts/gen_BeamToVD101.sh (hunk 1 above)
Comment (PawelPlesniak) [reply Scripts/gen_BeamToVD101.sh]: Thanks for catching that Rob. That was used for debugging which is now resolved. I have pushed the change.

**Review by kutschke — APPROVED** [review]
Thanks for looking after the comment out code.

**Comment by kutschke** [comment]
I don't believe that these changes have CI coverage.  So I am not rerunning the CI.

**Pull request merged**